src/material-moment-adapter/adapter/moment-date-adapter.ts

```diff
@@ -241,7 +248,14 @@ export class MomentDateAdapter extends DateAdapter<Moment> {
   }
 
   /** Creates a Moment instance while respecting the current UTC settings. */
-  private _createMoment(...args: any[]): Moment {
-    return (this._options && this._options.useUtc) ? moment.utc(...args) : moment(...args);
+  private _createMoment(
+    date: MomentInput,
+    format?: MomentFormatSpecification,
+    locale?: string,
+  ): Moment {
+    const strict = this._options ? this._options.strict : undefined;
```

Comment:
It looks like the `this._options` is used in a few places. Consider moving into a variable since it should minify a little better.

Reply:
I've attempted to do so. Let me know if it needs further change since I'm not sure this is what you requested.

Reply:
@crisbeto, is the refactor as you requested?